### PR TITLE
Updated macOS docs

### PR DESCRIPTION
**Pull request by RDIL**
I promise I’m not farming, just fixing grammar stuff

### CODE REVIEW

File: docs/guide/macOS.md [hunk 1]
```diff
@@ -1,27 +1,27 @@
 # macOS Virtual Machines
 
-It is possible to run macOS Virtual Machines the same way one can run [Linux containers](linux.md) on macOS Community Cluster. 
+It is possible to run macOS Virtual Machines the same way one can run [Linux containers](linux) on macOS Community Cluster.
```
Comment (fkorotkov) [inline docs/guide/macOS.md]: `.md` links are OK. There is some shenanigans with non-relative links though. See #145

Comment (RDIL) [reply docs/guide/macOS.md]: Okay

Comment (RDIL) [reply docs/guide/macOS.md]: fixed

File: docs/guide/macOS.md [hunk 2]
```diff
@@ -8,20 +8,20 @@ osx_instance:
...
-* `high-sierra-base` - vanilla macOS with Brew and Command Line Tools pre-installed.
+* `high-sierra-base` - vanilla macOS with [Homebrew](https://brew.sh) and Command Line Tools pre-installed.
 * `high-sierra-xcode-9.4.1` and `high-sierra-xcode-10.0` - based on `high-sierra-base` with Xcode and couple other packages pre-installed: `cocoapods`, `fastlane`, `rake` and `xctool`.
 
-Please refer to [`osx-images`](https://github.com/cirruslabs/osx-images) repository on how the images were built and
+Please refer to the [`osx-images`](https://github.com/cirruslabs/osx-images) repository on how the images were built and
 don't hesitate to [create issues](https://github.com/cirruslabs/osx-images/issues) if current images are missing something.
 
 !!! info "Underlying Technology"
-    Under the hood Cirrus CI is using [Anka Cloud hosted on MacStadium](supported-computing-services.md#anka) for
```
Comment (fkorotkov) [inline docs/guide/macOS.md]: Here too. Sorry. 😅

Comment (RDIL) [reply docs/guide/macOS.md]: fixed!

**Pull request merged**